### ISSUE TITLE
Week 6 team homework is released

### ISSUE THREAD

**Issue by doubleshow** [bigdatahci2015/forum#26]
Please fetch and merge as usual.

We are still finalizing the individual mongodb query drills and will publish them soon. Thanks for your patience.

**Comment by kjblakemore** [comment]
Are the individual mongodb query drills ready to be pulled?

**Comment by kalharbi** [comment]
@kjblakemore yes. Also, please use the missing db & collection names reported in #29
